index.md

```diff
@@ -25,3 +30,5 @@ Paragraph of introductory material.
 *   [Reference](reference.html)
 *   [Discussion](discussion.html)
 *   [Instructor's Guide](instructors.html)
+
+[swc-workshop-template-rendered]: http://swcarpentry.github.io/workshop-template/index.html
```

Comment:
I'd prefer https://swcarpentry.github.io/workshop-template/ for more security and a shorter URL.  Although it looks like we need to use HTTPS or a local image for the SWC banner in the workshop-template to make that work cleanly.